hopp/simulation/technologies/wind/floris.py

```diff
@@ -168,15 +176,35 @@ def value(self, name: str, set_value=None):
         else:
             return self.__getattribute__(name)
 
-    def set_floris_value(self, name, value):
+    def set_floris_value(self,name,value):
+        """Set value of FlorisModel object using the `set` function.
+
+        Args:
+            name (str): name of parameter to update.
+            value (any): value to assign to specified `parameter`.
+        """
         if value is not None:
             self.fi.set(**{name:value})
+    
+    def set_floris_param(self,param,value):
+        """Set parameter of FlorisModel object using the `set_param` function.
 
-    def set_floris_param(self, param, value):
+        Args:
+            param (str): name of parameter to update.
```

Comment:
This should align with the FLORIS `set_param`, which takes a list of strings, so it's not just a single name, unless it's a length-1 list.